src/generator/gentime/settingsSingleton.ts

```diff
@@ -83,6 +83,14 @@ export namespace Gentime {
     },
   })
 
+  /**
+   * Define runtime settings for Nexus Prisma.
+   *
+   * @example
+   *
+   *   changeSettings({ projectIdIntToGraphQL: 'Int' })
```

Comment:
Make the example runnable, so e.g. show how this is imported etc.

Reply:
You can steal one of the examples from https://github.com/prisma/nexus-prisma#runtime-settings.